examples/extend-graphql-schema-nexus/nexus/types/Post.ts

```diff
@@ -1,46 +0,0 @@
-/* -- NOTES --
-
-This example demonstrates using the prisma client (from Keystone's context)
-to return posts with a custom nexus-specified type.
-
-Ideally, we could tell Nexus about the GraphQL schema Keystone generates and
-not need to generate the output type again here, but that needs some more
-research (and maybe a plugin for nexus?)
-*/
-
-import { extendType, intArg, list, stringArg, nonNull, objectType } from 'nexus';
-
-export const NexusPost = objectType({
-  name: 'NexusPost',
-  definition(t) {
-    t.string('id');
-    t.string('title');
-    t.string('status');
-    t.string('content');
-  },
-});
```

Comment:
This wasn't actually required, we needed to use https://github.com/graphql-nexus/nexus/pull/983